cs/remote/src/FASTER.common/WireFormat.cs

```diff
@@ -23,6 +23,10 @@ public enum WireFormat : byte
         /// Similar to DefaultVarLenKV but with WebSocket headers (binary)
         /// </summary>
         WebSocket = 2,
+        
+        DarqSubscribe,
```

Comment:
are these really new wire format level items? and DARQ doesnt seem like somthing to be exposed in this level.

Reply:
Same as above for MessageType